converter.py

```
@@ -0,0 +1,156 @@
+        pass
+
+    def _format_lines(self):
+        for index, line in enumerate(self._lines):
+            self._lines[index] = line.strip("\n")
+
+
+class SCVHandler(Converter):
+    def __init__(self, input_file, output_file):
+        self._input_file = input_file
+        self._output_file = output_file
+        self._lines = None
+        self._titles = None
+        self._objects = self.read()
+        self._converted = self.convert()
+
+    def convert(self):
+        result_string = ""
+        if len(self) > 1:
+            result_string += "[ \n"
+            for obj in self:
+                result_string += "{ \n"
+                for key, value in obj.items():
+                    result_string += f'"{key}": "{value}", \n'
+                result_string = result_string[0:-3] + " \n"
+                result_string += "}, \n"
+            result_string = result_string[0:-3] + " \n"
+            result_string += "] \n"
+        else:
+            for obj in self:
+                result_string += "{ \n"
+                for key, value in obj.items():
+                    result_string += f'"{key}": "{value}", \n'
+                result_string = result_string[0:-3] + " \n"
+                result_string += "} \n"
+        return result_string
+
+    def write(self):
+        with open(self._output_file, "w") as file:
+            file.write(self._converted)
+
+    def read(self):
+        super().read()
+        self._titles = self.parse_titles()
+        data = []
+        for index, line in enumerate(self._lines):
+            if index > 0:
+                object_ = {}
+                coll_patterns = self._find_collection_patterns(line)
+                if not coll_patterns:
+                    for i in range(0, len(line.split(","))):
+                        try:
+                            object_.update({self._titles[i]: line.split(",")[i]})
+                        except IndexError:
+                            print("Index out of range")
```

Raise a self-explanatory created exception instead of print. So